### PR TITLE
[GEOS-9655] WMTS layer configuration page doesn't show all the native srs in getCapabilties document

### PR DESCRIPTION
**Pull request by taba90**
Jira ticket https://osgeo-org.atlassian.net/browse/GEOS-9655

This pull request makes possible to select from the find button in the layer configuration page, the other native srs when configuring a WMTS layer.
## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [x] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [x] PR for bug fixes and small new features are presented as a single commit
- [x] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [x] The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it.
- [x] New unit tests have been added covering the changes
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by travis-ci after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates (screenshots, text)
- [ ] Commits changing the REST API, or any configuration object, should check if the REST API docs (Swagger YAML files and classic documentation) need to be updated.

### CODE REVIEW

File: src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java [hunk 1]
```diff
@@ -360,11 +356,16 @@ private String getActualNativeSRSCode(ResourceInfo resourceInfo) {
         return "";
     }
 
-    private List<String> getOtherSRSFromWFSOrWMS(ResourceInfo resourceInfo) {
+    private List<String> getOtherSRSFromWFSOrWMSOrWMTS(ResourceInfo resourceInfo) {
```
Comment (aaime) [inline src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java:359]:
```suggestion
    /**
     * Returns a list of alternative SRS for resources that support multiple ones, e.g., WMS, WFS, WMTS cascaded layers
     */
    private List<String> getOtherSRS(ResourceInfo resourceInfo) {
```

File: src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java [hunk 2]
```diff
@@ -360,11 +356,16 @@ private String getActualNativeSRSCode(ResourceInfo resourceInfo) {
...
         // first check if its WFS-NG
         List<String> otherSRS = DataStoreUtils.getOtherSRSFromWfsNg(resourceInfo);
```
Comment (aaime) [inline src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java:361]: I'd go and check if it's a FeatureTypeInfo here.

File: src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java (hunk 1 above)
Comment (taba90) [reply src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java]: fixed

File: src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java (hunk 2 above)
Comment (taba90) [reply src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.java]: fixed

**Comment by taba90** [comment]
thanks @aaime your feedbacks have been applied

**Pull request merged**